apps/website-25/src/components/homepage/CommunitySection/TestimonialSubSection.tsx

```diff
@@ -0,0 +1,58 @@
+import { SectionHeading } from '@bluedot/ui';
+import { SlideList } from '@bluedot/ui/src/SlideList';
+
+export type Testimonial = {
+  quote: string;
+  name: string;
+  role: string;
+  imageSrc: string;
+};
+
+type TestimonialSubSectionProps = {
+  testimonials: Testimonial[],
+  title?: string,
+};
+
+const TestimonialSection = ({
+  testimonials,
+  title,
+}: TestimonialSubSectionProps) => {
+  return (
+    <div className="testimonial-section pb-spacing-y">
```

Comment:
nit: possibly change the BEM classname to `testimonial-subsection`?

separately this reminds me I would like to discuss BEM classnames, and what value they are adding atm